src/components/raycaster.js

```diff
@@ -60,25 +63,50 @@ module.exports.Component = registerComponent('raycaster', {
   },
 
   /**
+   * Update list of objects to test for intersection once child is loaded.
+   */
+  refreshOnceChildLoaded: function (evt) {
+    var self = this;
+    var el = evt.detail.el;
+    if (!el) { return; }
+    if (el.hasLoaded) { this.refreshObjects(); } else {
+      el.addEventListener('loaded', function nowRefresh (evt) {
+        el.removeEventListener('loaded', nowRefresh);
+        self.refreshObjects();
+      });
+    }
+  },
+
+  /**
    * Update list of objects to test for intersection.
    */
   refreshObjects: function () {
+    var children;
     var data = this.data;
     var i;
-    var objectEls;
+    var len;
+    var objects;
+    var objectsAreEls = data.objects ? this.el.sceneEl.querySelectorAll(data.objects) : null;
```

Comment:
see @donmccurdy recommendations above -- basically, this.objects should have only object3D when done, so the first part is exactly that, and the second part grabs the child object3D values because the top-level group one is useless.